src/main/java/com/healthsync/ui/PatientScene.java

```diff
@@ -1,31 +1,426 @@
 package com.healthsync.ui;
 
+import com.healthsync.dao.AppointmentsDao;
+import com.healthsync.dao.MessagesDao;
+import com.healthsync.dao.PatientDao;
+import com.healthsync.dao.UserDao;
+import com.healthsync.entities.Appointments;
+import com.healthsync.entities.Messages;
+import com.healthsync.entities.Patient;
+import com.healthsync.entities.User;
+import javafx.beans.value.ObservableValue;
 import javafx.geometry.Insets;
 import javafx.geometry.Pos;
-import javafx.scene.control.Label;
-import javafx.scene.layout.Priority;
-import javafx.scene.layout.Region;
-import javafx.scene.layout.VBox;
+import javafx.scene.Scene;
+import javafx.scene.control.*;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Color;
+import javafx.scene.shape.Rectangle;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextAlignment;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.util.Callback;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Stack;
 
 public class PatientScene extends BaseScene {
-    public PatientScene() {
-        super(createContent());
+    private static final PatientDao patientDao = new PatientDao();
+    private static final AppointmentsDao appointmentsDao = new AppointmentsDao();
+    private static final UserDao userDao = new UserDao();
+    private static final MessagesDao messagesDao = new MessagesDao();
+    public PatientScene(User user, Stage stage) {
+        super(createContent(user, stage));
     }
 
-    private static Region createContent() {
-        // THESE ARE SUPER BASIC BOXES WITH A TITLE,
-        // PLEASE GUT THEM WHEN YOU CREATE THE UI
-        VBox content = new VBox();
-        content.setAlignment(Pos.CENTER);
-        content.setSpacing(30);
-        content.setPadding(new Insets(20, 40, 20, 40));
+    private static Region createContent(User user, Stage stage) {
+        Patient patient = patientDao.getPatientById(user.getUserId());
+        DateFormat dateFormat = new SimpleDateFormat("MM-dd-yyyy");
```

Comment:
Does this match other parts of the code?

Reply:
The date format? Yes I took it from one of the other files, I think FindPatientScene. But lmk if you were referring to something else?